app/src/main/java/com/palone/planahead/screens/taskEdit/ui/taskTypeSelection/typeProperties/TypeProperties.kt

```diff
@@ -59,6 +61,7 @@ fun TypeProperties(modifier: Modifier = Modifier, viewModel: TaskEditViewModel)
         DateAndTimeDialog(onFinish = {
             viewModel.updateDateAndTime(it)
             shouldShowDateAndTimeDialog.value = false
-        })
+        }
```

Comment:
Actually, in case of anonymous delegate like that I think both `}` and `)` should be in a single line.

Reply:
🤦

Reply:
Sorry ;)